src/migrations/create.rs

```diff
@@ -978,13 +1004,48 @@ fn get_input(req: &RequiredUserInput) -> Result<String, anyhow::Error> {
     }
 }
 
+fn get_input_explicit(req: &RequiredUserInput) -> Result<String, anyhow::Error> {
+    let prompt = "Conversion expression:";
```

Comment:
Maybe we have a more detailed message here (use your judgement) -- like 

* `Please specify an EdgeQL expression that will be used to populate the existing optional fields in the database to become "required"`

* or `Please specify an EdgeQL expression that will be used to change existing data in property "foo" from "std::int64" to "std::str"`

* etc?